src/components/FlawForm.vue

```diff
@@ -169,16 +171,63 @@ const toggleMitigation = () => {
   }
 };
 
+const affectedOfferingsComp = ref<InstanceType<typeof AffectedOfferings> | null>(null);
+const referencesComp = ref<InstanceType<typeof IssueFieldReferences> | null>(null);
+const acknowledgmentsComp = ref<InstanceType<typeof IssueFieldAcknowledgments> | null>(null);
+
+const expandFocusedComponent = (parent_uuid: string) => {
+
+  // Expand Affect (affect, affect CVSS, tracker)
+  let trackers: ZodTrackerType[] = ([] as ZodTrackerType[]).concat(
```

Comment:
Why `let` instead of `const` if the variable is not being reassigned? (same for the rest of the function)

Reply:
This is a very good question. I would have swear that linter used to report these. @C-Valen @superbuggy @tboyetteredhat any idea why such thing is not complained about by the linter? :thinking:

Reply:
It looks it wasn't (there are many cases). Covered on this:
https://github.com/RedHatProductSecurity/osim/pull/253

Reply:
@C-Valen thank you for adding those rules! Will change the code accordingly.